File: src/main/java/fr/lewon/web/bot/entities/input/stat/ActionUpgradeStat.java
package fr.lewon.web.bot.entities.input.stat;

import fr.lewon.bot.http.body.urlencoded.FUEMember;
import fr.lewon.web.bot.entities.input.Action;

public class ActionUpgradeStat extends Action {

	@FUEMember
	private Integer carac;
	
	public ActionUpgradeStat(Integer carac) {
		super("Hero", "update_stats");
		this.carac = carac;
	}

}

File: src/main/kotlin/fr/lewon/bot/hh/entities/input/stat/ActionUpgradeStat.kt
package fr.lewon.bot.hh.entities.input.stat

import com.fasterxml.jackson.annotation.JsonProperty
import fr.lewon.bot.hh.entities.input.Action

class ActionUpgradeStat(carac: Int) : Action("Hero", "update_stats") {

    @JsonProperty
    val carac: Int = carac
}